yajsapi/executor/ctx.ts

```diff
@@ -20,55 +22,59 @@ export class CommandContainer {
   getattr() {
     const self = this;
     return {
-          get(target, name) {
+      get(target, name) {
         if (target[name] !== undefined) {
           return target[name];
         }
         const newFunction = function () {
           let _arguments = {};
-          let args = arguments[0] || {};
+          const args = arguments[0] || {};
           for (const [key, value] of Object.entries(args)) {
             _arguments = {
               ..._arguments,
               [key.startsWith("_") ? key.slice(1) : key]: value,
             };
           }
-          let idx = self._commands.length;
+          const idx = self._commands.length;
           self._commands.push({ [name]: _arguments });
           return idx;
         };
         return new Proxy(newFunction, {
           apply: function (target, thisArg, argumentsList) {
-            return target.apply(thisArg, argumentsList);
+            return target.apply(thisArg, argumentsList as []);
```

Comment:
is this a ts's way of saying "argumentsList" is an array?

Reply:
Yes exactly